notebooks/util/utils.py

```diff
@@ -0,0 +1,88 @@
+def create_migration_workflow(
+    gcs_input_path: str, gcs_output_path: str, project_id: str, 
+    bq_dataset: str, default_database: str, source_dilect: str, bq_region: str,
+    obj_name_mapping: str = None) -> None:
+    """
+    This function uses BQ translation API to convert DDLs/SQLs from different sources to BQ
+
+    Args:
+        gcs_input_path (string): GCS location where source DDLs are available
+        gcs_output_path (string): Output GCS location
+        project_id (string): Project ID
+        bq_dataset (string): BQ Dataset ID to be added in the final DDL
+        default_database (string): Project ID to be added in the final DDL
+        source_dilect (string): Can be hive|redshift|netezza|teradata|synapse|mysql|oracle|postgresql|presto|sparksql|SQLserver|vertica
+        bq_region: Region of BQ Dataset
+        obj_name_mapping: The mapping of objects to their desired output names in list form.
+
+    Returns:
+        name (string): Full name of the Migration Job
+        state (string): Job Run State
+
+    """
+    from google.cloud import bigquery_migration_v2
```

Comment:
Can you import from outside the function?

Reply:
Done.